src/libais/ais.h

```diff
@@ -2433,6 +2433,335 @@ class Ais27 : public AisMsg {
 };
 ostream& operator<< (ostream &o, const Ais27 &msg);
 
+
+// SSW Satellite Ship Weather 1-Slot Version
+class Ais8_367_23 : public Ais8 {
+ public:
+  int version;
+
+  int utc_day;
+  int utc_hour;
+  int utc_min;
+
+  AisPoint position;  // 25, 24 bits
+  int pressure;  // hPa
+  float air_temp;  // C
+  int wind_speed;  // knots
+  int wind_gust;  // knots
+  int wind_dir;  // degrees
+  int spare;
+
+  Ais8_367_23(const char *nmea_payload, const size_t pad);
+};
+ostream& operator<< (ostream &o, const Ais8_367_23 &msg);
+
+//
+// SSW Satellite Ship Weather 1-Slot Version
+class Ais8_367_24 : public Ais8 {
```

Comment:
Let's start off with a PR that is just Ais8_367_23 and do each message in tern so that we can carefully go through each message on its own.